src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Controllers/AdminController.cs

```diff
@@ -559,10 +561,15 @@ private void ValidateModel(ElasticIndexSettingsViewModel model)
             }
         }
 
-        private void PopulateMenuOptions(ElasticIndexSettingsViewModel model)
+        private async Task PopulateMenuOptionsAsync(ElasticIndexSettingsViewModel model)
         {
-            model.Cultures = CultureInfo.GetCultures(CultureTypes.AllCultures)
-                .Select(x => new SelectListItem { Text = $"{x.Name} ({x.DisplayName})", Value = x.Name });
+             var supportedCultures = await _localizationService.GetSupportedCulturesAsync();
```

Comment:
@sebastienros I'm not sure this is appropriate to list only supported cultures. This means that now the ElasticSearch indices culture can only be supported cultures that are adding in the Admin UI. This means that I need to add translations to each of these supported cultures while I could have been able to loosely couple a culture with an index before.

Reply:
As an example of that I could have a website that is majorly written in french but has some content in english which I would want to index in a different indice if required.

Reply:
> This means that I need to add translations to each of these supported cultures while I could have been able to loosely couple a culture with an index before.

How this different from getting all cultures?

Reply:
They come from the framework instead of the DB. And the ASP.Net localization middleware will support these cultures if they are added. if you just list cultures that are in the framework it doesn't do anything else than that.

Reply:
I knew what you mentioned :)

What I mean how the change that I made is different for the Elastic search module?

Reply:
Different in what matter? I'm just saying that we should not use supported cultures only for this and also Lucene or any other Search index configuration.

Reply:
Frankly, I didn't understand why using the supported cultures in this case is not preferred!! maybe @sebastienros understand what you are referring too

Reply:
You could have a website that is majorly written in english. You would have only "en" as a supported culture. 
But your website has content written in french for a specific content type... then you would not want to support "fr" but only index that content type for the "fr" culture in a custom handler...